packages/settings-library/setup.cfg

```diff
@@ -14,5 +14,6 @@ universal = 1
 # Define setup.py command aliases here
 test = pytest
 
-[tool:pytest]
-asyncio_mode = auto
+# NOTE: uncomment when pytest-asyncio is added in requirements
+# [tool:pytest]
+# asyncio_mode = auto
```

Comment:
shall we add this everywhere and remove from the pytest cmds?

Reply:
It is already added everywhere